### PR TITLE
Refact: add localStorage helper.

### PR DESCRIPTION
**Pull request by fdrissi**
- localStorage set, get, remove and clear helper.
issue: #14

### CODE REVIEW

File: src/helpers/storage.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const set = (key, value) => {
+  window.localStorage.setItem(key, value)
+}
```
Comment (smakosh) [inline src/helpers/storage.js]: You could turn these into one line functions like so:
```js
const set = (key, value) => window.localStorage.setItem(key, value)
```

**Review by smakosh — CHANGES_REQUESTED** [review]
Thanks @fdrissi it looks good to me, just added few comments for simplicity, btw do you have Prettier extension installed because normally it should format that on save and do as I requested?

Could you also raise the same PR on the `typescript` branch?

File: src/helpers/storage.js (hunk 1 above)
Comment (fdrissi) [reply src/helpers/storage.js]: Hey @smakosh 👋 , prettier sometimes doesn't work in my vscode.
I will check that later tonight, and for sure I will add the ts version.

**Review by smakosh — APPROVED** [review]
LGTM

**Pull request merged**